### PR TITLE
Mk challange

### PR DESCRIPTION
**Pull request by mohsenkazemi**
added rfactor computation to resolution_fsc

### CODE REVIEW

File: software/em/xmipp/applications/programs/resolution_fsc/resolution_fsc_main.cpp [hunk 1]
```diff
@@ -82,6 +82,11 @@ class ProgResolutionFsc : public XmippProgram
         addParamsLine("   [--dont_apply_geo]        : for 2D-images: do not apply transformation stored in the header");
         addParamsLine("   [--do_dpr]                : compute dpr, by default only frc is computed");
         addParamsLine("   [--max_sam <max_sr=-1>]   : set fsc to 0 for frequencies above this one (Angstrom), -1 -> all fequencies");
+        addUsageLine ("             --max_sam = 10A -> frequencies higher than 0.1 A^-1 =0");
```
Comment (delarosatrevin) [inline software/em/xmipp/applications/programs/resolution_fsc/resolution_fsc_main.cpp]: This should be also the addParamsLine and use the : for mark as help.
Check in other following lines.

**Comment by delarosatrevin** [comment]
@juannavascalvente , I reviewed the code changes with @cossorzano and seems to be fine. Could you pass the tests (make sure you passed the newly added test in this branch) and accept the PR?

**Comment by delarosatrevin** [comment]
@juannavascalvente, can you double check? Then you can made the merge.

**Comment by juannavascalvente** [comment]
Test passed and gold has been updated with new test results.

**Pull request merged**